lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.hmpps.sqs
+
+import com.amazonaws.services.sqs.AmazonSQS
+import com.amazonaws.services.sqs.model.DeleteMessageRequest
+import com.amazonaws.services.sqs.model.Message
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest
+import org.springframework.stereotype.Service
+
+@Service
+class HmppsQueueService {
+
+  private val hmppsQueues: MutableList<HmppsQueue> = mutableListOf()
+
+  fun registerHmppsQueue(hmppsQueue: HmppsQueue) {
+    hmppsQueues += hmppsQueue
+  }
```

Comment:
Slightly early optimisation to cater for multiple queues, but we know that is definitely a requirement so worth getting it in now.